### PR TITLE
Add inline doc comments to the Web SDK template

### PR DESCRIPTION
**Pull request by Im-Madhur-Gupta**
## What does this PR do?

This PR adds inline comments to the web SDK template. These comments will be visible in IDEs.

## Test Plan

I generated the SDK from the web template after making changes into it using `example.php`. This verified that the comments were showing up correctly.

## Related PRs and Issues

- https://github.com/appwrite/sdk-generator/issues/679
- https://github.com/appwrite/sdk-generator/pull/671

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes, I have.

### CODE REVIEW

**Comment by Im-Madhur-Gupta** [comment]
I have added the comments by referencing the [PR](https://github.com/appwrite/sdk-generator/pull/671) for doc comments for Dart & Flutter as mentioned in the original [issue](https://github.com/appwrite/sdk-generator/issues/679) here. Also, there were some comments missing for the client.ts.twig file (for RealtimeResponse, RealtimeRequest, RealtimeResponseEvent etc), I have added these from my end.

Please lmk in case I have missed anything, I'll add it asap.

**Review by PineappleIOnic — APPROVED** [review]

**Comment by lohanidamodar** [comment]
Hi @Im-Madhur-Gupta great work, we would like to proceed with this, if you are still interested to complete this, can you please sync this with latest changes. No worries if you are no longer working on this just let us know. Thank you.

**Comment by Im-Madhur-Gupta** [comment]
Sure @lohanidamodar, let me sync my branch with main.

**Comment by Im-Madhur-Gupta** [comment]
@lohanidamodar I have merged my branch with master, pls check. Thanks.

**Comment by gewenyu99** [comment]
Hey there! There were a lot of big PRs during this Hacktoberfest, and we wanted to give everyone ample time to collaborate with our engineering team. If you were able to merge your PRs during October, amazing. If it’s still not merged, don’t worry about it either. Either way, we’ve got your Hacktoberfest swag minted and ready to ship. 

Please comment with your Discord username here so we can contact you about your shipping information to deliver your Hacktoberfest swag.

**Comment by Im-Madhur-Gupta** [comment]
Hi @gewenyu99, thank you, here is my Discord username: immadhurgupta

File: templates/web/README.md.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # {{ spec.title }} {{sdk.name}} SDK
 
 ![License](https://img.shields.io/github/license/{{ sdk.gitUserName|url_encode }}/{{ sdk.gitRepoName|url_encode }}.svg?style=flat-square)
-![Version](https://img.shields.io/badge/api%20version-{{ spec.version|url_encode }}-blue.svg?style=flat-square)
+![Version](https://img.shields.io/badge/api%20version-{{ spec.version | split('.') | slice(0,2) | join('.') ~ '.x' | url_encode }}-blue.svg?style=flat-square)
```
Comment (lohanidamodar) [inline templates/web/README.md.twig:4]: @Im-Madhur-Gupta Where did this change come from?

Comment (Im-Madhur-Gupta) [reply templates/web/README.md.twig:4]: Hi, facing some medical issues currently, will get back to you in a couple of days.

Comment (Im-Madhur-Gupta) [reply templates/web/README.md.twig]: @lohanidamodar Apologies for the delay in reply, I added this code while I was working on the ticket to fix some issue with the version batch but as I can see now we don't need it, hence I have reverted this change.

**Review by lohanidamodar — APPROVED** [review]

**Pull request merged**